core/src/components/textarea/textarea.md.scss

```diff
@@ -29,3 +32,18 @@
 .textarea-bottom .counter {
   letter-spacing: 0.0333333333em;
 }
+
+/**
+ * This makes the label sit above the textarea.
+ */
+:host(.textarea-label-placement-stacked) .label-text-wrapper,
+:host(.has-focus.textarea-label-placement-floating) .label-text-wrapper,
+:host(.has-value.textarea-label-placement-floating) .label-text-wrapper {
+  @include transform(translateY(6px), scale(#{$textarea-md-floating-label-scale}));
+
+  /**
+    * Label text should not extend
+    * beyond the bounds of the textarea.
+    */
+  max-width: calc(100% / #{$textarea-md-floating-label-scale});
```

Comment:
The input changes I did use the same scale for both iOS and MD inputs. Do we need to do the same thing here/do something differently for the input?

Reply:
This may be up to the team's interpretation. The floating label scale for the MD design is `0.75`. Ionic v6 has a floating label scale of `0.82` strictly for iOS. iOS does not have a floating label design implementation in the human interface guidelines, so this was a custom design decision at some point in the history of Ionic. 

Do we either:
- Align consistently against MD spec for both mode (would require a change to this PR).
- Maintain the `0.82` for iOS and `0.75` for MD, in v7 (would require a change to input).

Reply:
I'm always in favor of consistency, so I'd vote for making the scales the same in this PR.